### ISSUE TITLE
Select SAGA version during initialization

### ISSUE THREAD

**Issue by gisma** [stevenpawley/Rsagacmd#2]
@stevenpawley 
first of all big thumb up and clapping hands for this project. I am dealing quite a while with SAGA, RSAGA and all the mess around it. From personal communication I was told  that there will be an update of  RSAGA... However I did not want to wait and I am surprised that I just found you by a brute force web search dealing in the broadest sense with bridging  `R` and `SAGA`. 
During my first tests your package works out well I even can switch the installations using a simple function `findSAGA` from my [link2GI](https://github.com/gisma/link2GI) package to do so. 
Maybe using my tool is sufficient but it would probably a nice build-in feature of `Rsagacmd` as you only need to extend the approach how to deal with the multiple installations found. 
nevertheless thanks for you efforts!
cheers Chris

**Comment by stevenpawley** [comment]
Hi Chris, thanks for your interest! Currently I have only implemented a crude built-in method to search for the SAGA-GIS installation, so there is certainly scope for improvement in comparison to your findSAGA function. I started writing Rsagacmd, not necessarily due to limitations with RSAGA, but because I needed a way to script SAGA-GIS for workflows where I can rarely remember each tool's syntax due to the inconsistencies with SAGA-GIS arguments (i.e., INPUT, DEM, ELEV, ELEVATION...). Over the past few days I have been trying to make Rsagacmd play more nicely with pipes, so that unspecified arguments that represent non-optional outputs from each tool are automatically written to tempfiles, thus making it possible to do things like:

flow_acc = dem_clipped %>%
  saga$ta_preprocessor$Fill_Sinks_Wang_Liu(MINSLOPE = 0.01) %>%
  {saga$ta_hydrology$Flow_Accumulation_Top_Down(ELEVATION = .$FILLED)}
(with caveats that many SAGA-GIS tools produce multiple outputs)

I'm hoping to add some other features such using an option of using grid tiling for any relevant function to make it more convenient to process large data sets that can't possibly fit into memory.
